test/DateTime.cpp

```diff
@@ -28,4 +48,17 @@ unittest(yearMonthAsPath) {
   assertEqual("/2020/09", buffer);
 }
 
+unittest(setAsCurrent) {
+  DateTime_TC jan(2021, 1, 15, 1, 48, 24);
+  DateTime_TC dob(2001, 2, 20, 3, 50, 10);
```

Comment:
To minimize revealing info, let's just use `jan` as "now" and adjust following lines of asserts.